extensions/ffghdfh/indexedDB.js

```diff
@@ -0,0 +1,222 @@
+// Name: Indexed Database
```

Comment:
Credit yourself here! You made the extension! :)

Reply:
done! i added my discord username!